projects/dev-app/src/app/grid/grid-demo.component.ts

```diff
@@ -88,7 +92,6 @@ export class GridDemoComponent implements OnInit, AfterViewInit {
       header: this.translate.stream('city'),
       field: 'city',
       minWidth: 150,
-      class: 'text-center',
```

Comment:
Please merge the lastest code.

Reply:
Weird, I forked main branch yesterday and it didn't fetched that change for some reason